### ISSUE TITLE
[NPC] Foe Reaper 5000 bugged

### ISSUE THREAD

**Issue by phonkala** [SallyWhitemane/Maelstrom#41]
**Description:**
Foe Reaper 5000 seems to be quite bugged, most of the time it just keeps resetting its health to full nonstop and keeps players in combat. First I thought this is issue with stealth (https://github.com/SallyWhitemane/Maelstrom/issues/40) but have now tested it 3 more times, never going into stealth and not even attacking it before waiting 5-10 sec. All the last 3 times it has been bugged. However, once I did somehow manage to kill it, no idea how.

Also the other "small" reapers that you need to kill in the boss room before attacking boss are bugged. After killing them they do go friendly, but they reset their health to full (hp should remain) and you can not use them (should be able to use them with their remaining health to help kill other stuff in the room). This isn't game-breaking though, but the boss getting bugged is.

**Comment by ghost** [comment]
Please, retest it.

**Comment by phonkala** [comment]
Still same issue. I never went into stealth during boss event, and Foe Reaper 5000 remained standing still, healing to full all the time when attacked.

Also the small reapers in the room are still as they were when reporting this issue (so not fixed), but I dunno if it's even worth fixing those as they are by no means necessary to kill the boss.

**Comment by phonkala** [comment]
Managed to kill the boss this way:
- After killing the adds, I ran straight to hug the boss while he is doing the rp to start the boss event
- During that rp, he did the initial shock of lightnings/electricity, which instantly killed me (dunno if it should but thats another thing)
- That way the boss didn't ever really engage combat, as I died before that
- Then I just ran in and killed the boss, it was not and didn't get bugged anymore. Though it did wait 10-15sec before doing anything at start, while I was attacking it.
- Weirdly the "small" reapers didn't respawn at all this way. They do respawn, if I kill them and the boss is then bugged, forcing me to run out and back in again.
